core/exporter/encoders/json.go

```diff
@@ -457,14 +457,20 @@ func mapPortList(writer *jwriter.Writer, ports *[]*sfgo.Port) {
 	writer.RawString("ports")
 	writer.RawString(QUOTE_COLON)
 	writer.RawByte(BEGIN_SQUARE)
-	for _, p := range *ports {
+	for i, p := range *ports {
+		writer.RawByte(BEGIN_CURLY)
 		writeIntField(writer, "port", p.Port)
 		writer.RawByte(COMMA)
 		writeIntField(writer, "targetport", p.TargetPort)
 		writer.RawByte(COMMA)
 		writeIntField(writer, "nodeport", p.NodePort)
 		writer.RawByte(COMMA)
 		writeStrField(writer, "proto", p.Proto)
+		if i<len(*ports){
+			writer.RawByte(END_CURLY_COMMA)
```

Comment:
Since END_CURLY_COMMA is two characters, I think it should be:
```
writer.RawString(END_CURLY_COMMA)
```
RawByte will only write one byte I think..

Reply:
Yes, makes sense.

Reply:
I also fixed an off-by-one in the if condition that was causing the arrays to have a comma at the end of the JSON array. 
```
if (i + 1) < len(*ports)
```
I ran tests on some small and large stock SysFlow traces collected from k8s, and the JSON output seems to be correct. Thanks for this PR, @VishwasSomasekhariah !